docs/Installation.md

```diff
@@ -128,6 +123,7 @@ To install the `mlagents` Python package, activate your virtual environment and
 run from the command line:
 
 ```sh
+pip3 install torch -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
should we mention this is only for windows?

Reply:
It doesn't hurt on the other OS's, but I agree it's clearer if we separate windows. Updated the instructions - let me know what you think